BUILDING.md

```diff
@@ -766,23 +766,42 @@ to enable FIPS using the configuration flag `--openssl-is-fips`.
 ### Configuring and building quictls/openssl for FIPS
 
 For quictls/openssl 3.0 it is possible to enable FIPS when dynamically linking.
-Node.js currently uses openssl-3.0.0+quic which can be configured as
-follows:
+If you want to build Node.js using openssl-3.0.0+quic, you can follow these
+steps:
+
+**clone OpenSSL source and prepare build**
 ```console
```

Comment:
`console` represents both commands (when the line starts with a `$`) and stdout/stderr. If you are just listing commands, use `base` instead.
````suggestion
```bash
````